about/license.md

```diff
@@ -1,10 +1,12 @@
 ---
-description: Strong Commitment to the Openness and Collaboration
+description: Strong commitment to openness and collaboration
```

Comment:
```suggestion
title: License
description: Fluent Bit license description
```

Reply:
We should add a `title`, and maybe update that description to match the page contents.